README.md

```diff
@@ -48,26 +46,37 @@ pip install -e .
 
 1. Ensure Docker is installed and running
 
-1. `docker run -p7687:7687 -p7474:7474 --env NEO4J_AUTH=neo4j/password --env=NEO4J_ACCEPT_LICENSE_AGREEMENT=yes neo4j:3.5.14-enterprise` (in another terminal session or with `-d` (detached) flag)
+2. Create a file for the GCP credentials in your HOME folder: `~/.secrets/gcp_credentials`
```

Comment:
Have you considered creating a setup script with the commands to create the directory and download the secret?

e.g.

```bash
mkdir ~/.secrets
aws secretsmanager get-secret-value --secret-id ingest/${ENV}/gcp-credentials.json  --region=us-east-1
```

Reply:
@amnonkhen The task was to remove this app from pypi. The above you are asking is not belongs to that task.